src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -1873,14 +1879,21 @@ async def _handle_action(self, request_id: Optional[str], data: RTVIActionRun):
             message = RTVIActionResponse(id=request_id, data=RTVIActionResponseData(result=result))
             await self.push_transport_message(message)
 
-    async def _send_bot_ready(self):
-        """Send the bot-ready message to the client."""
+    async def _send_bot_ready(self, about: Mapping[str, Any] = None):
+        """Send the bot-ready message to the client.
+
+        Args:
+            about: Optional information about the bot to include in the ready message.
+                   If left as None, the pipecat library and version will be used.
+        """
         config = None
         if self._client_version and self._client_version[0] < 1:
             config = self._config.config
+        if not about:
+            about = {"library": "pipecat-ai", "library_version": f"{lib_version('pipecat-ai')}"}
```

Comment:
Then use the pipecat version here:
```suggestion
            about = {"library": "pipecat-ai", "library_version": pipecat_version}
```